regexes/device/mobiles.yml

```diff
@@ -7527,6 +7586,32 @@ Zen:
   device: 'smartphone'
   model: '$1'
 
+# Lemhoov
+Lemhoov:
+  regex: 'Lemhoov'
+  device: 'smartphone'
+  models:
+    - regex: 'Lemhoov[ _-]([^/;]+) Build'
+      model: '$1'
+
+#Libra
+Libra:
+  regex: 'Libra[ _-]'
+  device: 'tablet'
+  models:
+    - regex: 'Libra[ _-](97) Build'
+      model: '$1'
+
+#MTC
+MTC:
+  regex: 'MTC[ _-](978|982|982O|982T)|SMART[ _]Race[ _]4G Build'
+  device: 'smartphone'
+  models:
+    - regex: 'MTC[ _-](978|982|982O|982T) Build'
```

Comment:
Guess the ` Build` here isn't required, as the global regex doesn't contain it as well.